onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1825,10 +1829,20 @@ def aten_dot(self: TFloat, tensor: TFloat) -> TFloat:
     return op.MatMul(self, tensor)
 
 
-def aten_dropout(input: TensorType, p: float, train: bool) -> TensorType:
-    """dropout(Tensor input, float p, bool train) -> Tensor"""
+@torch_op("aten::dropout")
+def aten_dropout(input: TFloat, p: float = 0.5, train: bool = True) -> TFloat:
+    # dropout(Tensor input, float p, bool train) -> Tensor
 
-    raise NotImplementedError()
+    rank_input = op.Size(op.Shape(input))
+    if rank_input == 0:
+        input = op.Reshape(input, op.Constant(value_ints=[-1]))
+
+    result, _ = op.Dropout(input, p, train)
```

Comment:
```suggestion
    result = op.Dropout(input, p, train)
```

Reply:
op.Dropout() return (result, mask), but aten_dropout only need result without mask.

Reply:
@gramalingam i think this is another example when eager mode differs with the function, as Dropout can have one or two outputs but eager mode returns two no matter what?

Reply:
[This](https://github.com/microsoft/onnx-script/blob/main/onnxscript/evaluator.py#L319) needs to be fixed.

Either a two-line short-term patch for specifically Dropout to indicate the expected number of outputs, or a longer term fix of invoking ONNX inference to actually figure this out. (The longer-term solution is a bit non-trivial.)

Reply:
(So, I recommend the short-term fix for immediate use.)